mockserver/src/main/groovy/pl/touk/mockserver/server/Mock.groovy

```diff
@@ -60,7 +60,7 @@ class Mock implements Comparable<Mock> {
 
     boolean match(Method method, MockRequest request) {
         boolean usesCondition = hasLimitedUses() ? usesLeft > 0 : true
-        return usesCondition && this.method == method && predicate(request)
+        return usesCondition && (this.method == method || this.method == Method.ANY) && predicate(request)
```

Comment:
Let's add some documentation for this behaviour and option

Reply:
Done.